src/components/TextInput/TextInput.tsx

```diff
@@ -4,6 +4,7 @@ import cx from 'classnames';
 type TextInputProps = {
   className?: string;
   describedBy?: string;
+  disabled?: boolean;
```

Comment:
And again (sorry, just making sure I catch them all!)... could we use `isDisabled`